schemas/com.google.analytics.measurement-protocol/app_tracking/jsonschema/1-0-0

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Google Analytics app tracking entity",
+  "self": {
+    "vendor": "com.google.analytics.measurement-protocol",
+    "name": "app_tracking",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+
+  "type": "object",
+  "properties": {
+    "applicationName": {
+      "type": ["string", "null"],
+      "maxLength": 100
+    },
+    "applicationId": {
+      "type": ["string", "null"],
+      "maxLength": 150
+    },
+    "applicationVersion": {
+      "type": ["string", "null"],
+      "maxLength": 100
+    },
+    "applicationInstallerId": {
+      "type": ["string", "null"],
+      "maxLength": 150
+    }
+  },
```

Comment:
since this seems to be modeling a `screenview` hit type, I'm wondering why there is no:

```json
"hitType": {
  "enum": [
    "sceenview"
  ]
}
```

like in the other schemas representing hit types (e.g. `item` or `page_view`)